### PR TITLE
feat: add QA agent + qa-testing skill

### PR DESCRIPTION
**Pull request by espennilsen**
## Summary

Phase 1 of the QA agent — agent definition + skill, no extensions needed.

### QA Agent (`agents/qa.md`)
- Tests running web apps via cmux_browser (Playwright under the hood)
- Structured workflow: Setup → Smoke Test → Functional → Edge Cases → Accessibility → Report
- 5-dimension grading (1-10): Functionality, Completeness, UX, Robustness, Accessibility
- PASS/FAIL verdict (PASS = all dimensions ≥ 6, no critical failures)
- Never modifies code — test-only with screenshot evidence
- axe-core injection for automated accessibility auditing
- Tools: read, bash, cmux_browser, cmux_split, cmux_read, cmux_send, cmux_close, cmux_list, cmux_notify

### QA Testing Skill (`skills/qa-testing/SKILL.md`)
- Step-by-step workflow with exact cmux_browser tool call examples
- axe-core injection snippet + result interpretation guide
- Lighthouse CLI template for performance auditing
- Full report template with grading rubric
- Edge case testing checklist (empty states, long text, XSS, rapid clicks, back button)
- Triggers: "test this", "QA this PR", "run QA", "check the app"

### Design basis
Based on Velma's research report (`qa-agent-design.md`), modeled on Anthropic's evaluator pattern (separate evaluator agent, explicit rubrics, PASS/FAIL with feedback).

Closes td-c3247c

### CODE REVIEW

File: skills/qa-testing/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+
+**Scoring accessibility:**
+| Violations | Score |
+|-----------|-------|
+| 0 violations | 10 |
+| 1-3 minor | 8-9 |
+| 1-3 serious | 6-7 |
+| 4-10 mixed | 4-5 |
+| 10+ or any critical | 2-3 |
+
```
[the remark this answers is not in the document]
Comment (espennilsen) [reply skills/qa-testing/SKILL.md:198]: Fixed — updated Grading Rubric Reference to match the detailed Step 5 accessibility scoring table (0, 1-3 minor, 1-3 serious, 4-10 mixed, 10+ or critical)

**Comment by espennilsen** [comment]
## ✅ Review feedback addressed

### Fixes applied (commit 546f1e0):

1. **🔧 agents/qa.md — Expanded accessibility rubric**
   - Added missing "1-3 minor violations" band to the Accessibility scoring row
   - Now consistent with SKILL.md's 5-band rubric (10 / 8-9 / 6-7 / 4-5 / 2-3)
   - Resolves the scoring gap for minor vs. serious violations

2. **🔧 skills/qa-testing/SKILL.md — Fixed invalid cleanup command**
   - Removed invalid `bash` prefix from Docker cleanup command in Step 8
   - Changed `bash cd /path/to/project && docker compose down` → `cd /path/to/project && docker compose down`
   - Now executes correctly without attempting to run `cd` as a bash script

### Already correct (no action needed):
- ✅ **Verdict rule** — Both files already use `< 6` for FAIL, which correctly catches scores of 5
- ✅ **SKILL.md accessibility scoring** — Step 5 table and Grading Rubric Reference are consistent

### Not addressed (by design):
- **axe-core CDN fallback** — CDN failure noted in docs with fallback suggestion; low priority for initial version
- **Explicit cleanup step** — Step 8 exists and is now fixed

Both markdown files are aligned and ready for merge. No TypeScript or executable code changes.
